docs/reference/migration/migrate_8_0/security.asciidoc

```diff
@@ -306,6 +306,42 @@ renamed to better reflect its intended use.
 *Impact* +
 Assign users with the `kibana_user` role to the `kibana_admin` role.
 Discontinue use of the `kibana_user` role.
+====
+
+[discrete]
+[[saml-realm-nameid-changes]]
+===== The default value of `nameid_format` setting has been removed.
+
+.The `nameid_format` setting of SAML realms has no default value.
+[%collapsible]
+====
+*Details* +
+Identity Providers can be either statically configured to release a `NameID`
+with a specific format, or they can be configured to try to conform with the
+requirements of the SP. The SP declares its requirements as part of the
+Authentication Request, using an element which is called the `NameIDPolicy`
+For the {es} SAML realm, `NameIDPolicy` is controlled by `nameid_format` in
+SAML realm settings.
+
+Previously, the default value for this setting was
+`urn:oasis:names:tc:SAML:2.0:nameid-format:transient` which meant that {es}
+would create SAML Authentication Requests that would require from the Identity
+Provider to release `NameID` with a transient format.
+The default value has now been removed. This means that {es} will be default
+create SAML Authentication Requests that do not put forward such requirements
+to the Identity Provider.
+
+If you want to retain the previous behavior, you can set `nameid_format`
+to `urn:oasis:names:tc:SAML:2.0:nameid-format:transient`.
```

Comment:
I wonder if we can remove the second sentence.
```suggestion
This default has been removed. To retain the previous default behavior, set
`nameid_format` to `urn:oasis:names:tc:SAML:2.0:nameid-format:transient`.
```

Reply:
I don't think it makes sense to remove this. This is the essence of what changes and what this change means.

Reply:
+1 to this change -- I'll include it in the next push.